### ISSUE TITLE
Expose gpu-driver-version for GPU configurability

### ISSUE THREAD

**Issue by diervo** [terraform-google-modules/terraform-google-kubernetes-engine#1787]
### TL;DR

Expose the ability to configure `gpu-driver-version` via terraform while using `terraform-google-modules/kubernetes-engine/google//modules/beta-public-cluster`.

Only `accelerator_count` and `accelrator_type` are avialable.

Seems that there is a big mismatch between some of the docs

### Expected behavior

Have the ability to customize the value per documentation:
https://cloud.google.com/kubernetes-engine/docs/how-to/gpus#terraform

Unclear how that API matches the module described in the documentation above.

### Observed behavior

Inability to install the drivers automatically. Ambiguity to understand the defaults.

### Terraform Configuration

```hcl
+ provider registry.terraform.io/hashicorp/google v4.84.0
+ provider registry.terraform.io/hashicorp/google-beta v4.84.0
+ provider registry.terraform.io/hashicorp/kubernetes v2.23.0
+ provider registry.terraform.io/hashicorp/random v3.5.1
```

### Terraform Version

```sh
Terraform v1.3.7 on darwin_arm64
```

### Additional information

It will be nice to know wether drivers are installed by default to not.

**Comment by Vhaos** [comment]
@diervo some good news hopefully 

Support for specifying a `gpu_driver_version` was added in https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1767 and looks scheduled for release in v29.0.0 (See https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1746)
